api/api/models/__init__.py

```diff
@@ -176,6 +178,25 @@ class Meta:
     creation_date: models.DateField = models.DateField(auto_now=True, editable=False)
 
 
+class Freelancer(models.Model):
+    """
+        Freelancer model
+
+        a mixin of manager & pentester that can do a little bit of both
+
+        auth -> one-to-one to Auth model
+        creation_date -> read-only field expressing creation date
+    """
+    class Meta:
+        verbose_name = 'Freelancer'
+        verbose_name_plural = 'Freelancers'
+        ordering = ['creation_date']
+
+    id: models.AutoField = models.AutoField(primary_key=True)
```

Comment:
why ?it's automatically generated by Django

Reply:
autocompletion things